src/cljs/main/broadfcui/page/workspaces_list.cljs

```diff
@@ -226,7 +226,10 @@
                                 (or (not public)
                                     (common/access-greater-than-equal-to? accessLevel "WRITER")))}
                   {:label "Public Workspaces"
-                   :predicate :public}]}
+                   :predicate :public}
+                  {:label "Featured Workspaces"
+                   :predicate (fn [workspace]
+                                (contains? (:featured-workspaces props) (ws-common/workspace->id workspace)))}]}
```

Comment:
This predicate is logically correct, but it has slowed down browser performance on the workspace list when switching tabs - to an extent that I can see it with the naked eye. Look at the `attach-table-data` function earlier in this file. I think we may need to pre-calculate a `featured?` key for the workspace during `attach-table-data`, and use that key in the predicate.